### ISSUE TITLE
Add ICES:SHIPC to http://schema.geolink.org/voc/identifiers#

### ISSUE THREAD

**Issue by sparkji** [ec-geolink/design#84]
Hi Bryce,
Please add ICES:SHIPC to http://schema.geolink.org/voc/identifiers#, we need it.
ICES:SHIPC (http://vocab.ices.dk/)  is SeaDataNet Ship and Platform Codes, we use them to identifier our vessels, like below:
V a Vessel
V hasIdentifier :I 
:I hasIdentifierScheme "http://schema.geolink.org/voc/identifiers#ICES:SHIPC"
:I  hasIdentifierValue <ICES:SHIPC:31A5>

Thanks,
Peng

**Comment by krisnadhi** [comment]
Hi @sparkji,

Is the code list you requested the same as the one here?
http://vocab.nerc.ac.uk/collection/C17/current/

**Comment by sparkji** [comment]
Yes. But I am not sure which property in GBO I could use  to directly link
platform instance in my graph with this vocabulary. Right now, I just
use "rdfs:seeAlso:, like P rdfs:seeAlso <
http://vocab.nerc.ac.uk/collection/C17/current/ 32D9>. It's not good
solution, using Idenifier could be more consistent, is right?

**Comment by krisnadhi** [comment]
Since http://vocab.nerc.ac.uk/collection/C17/current/32D9/ is actually an instance of a platform and we already have a Platform class, I can just adopt C17 code list into the base ontology as instances of Platform (in a separate OWL file, of course). I (or @amoeba ) can additionally add an appropriate identifier scheme for the C17 code list.

What you could do is to simply associate your vessel with the code list like so:
V a glbase:Vessel .
V owl:sameAs <http://vocab.nerc.ac.uk/collection/C17/current/32D9/>.

This is also in accordance with the consensus in last week's DC meeting that it's okay to use external vocabulary providers' URIs in the GBO triples.

**Comment by sparkji** [comment]
It sounds good.
